templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
+		  <h1>TrackCat</h1>
+        </div>
+        <div class="h2">
 		<p>A tool for tracking the progress of the kittens (and cat{s,z}) in out CodeCatz meetup group</p>
-	</div>
+        </div>
+    </div>
```

Comment:
You're still using spaces instead of tabs, please check the settings of your text editor (again, not a functionality problem, but we're trying to make all files look like they are part of the same project :) ).